lib/tutorials/en_US/auth.md

```diff
@@ -1,14 +1,115 @@
+
+## <a name="default" /> Default Strategy
+
+You may set a default strategy by using `server.auth.default()`.
+
+This method accepts one parameter, which may be either a string with the name of the strategy to be used as default, or an object in the same format as the route handler's [auth options](#route-configuration).
+
+Note that any routes added *before* `server.auth.default()` is called will not have the default applied to them. If you need to make sure that all routes have the default strategy applied, you must either call `server.auth.default()` before adding any of your routes, or set the default mode when registering the strategy.
+
+## <a name="route" /> Route Configuration
+
+Authentication can also be configured on a route, by the `options.auth` parameter. If set to `false`, authentication is disabled for the route.
+
+It may also be set to a string with the name of the strategy to use, or an object with `mode`, `strategies`, and `payload` parameters.
+
+The `mode` parameter may be set to `'required'`, `'optional'`, or `'try'` and works the same as when registering a strategy.
+
+If set to `'required'`, in order to access the route, the user must be authenticated, and their authentication must be valid, otherwise they will receive an error.
+
+If `mode` is set to `'optional'` the strategy will still be applied to the route but in this case the user does *not* need to be authenticated. Authentication data is optional, but must be valid if provided.
+
+The last `mode` setting is `'try'`. The difference between `'try'` and `'optional'` is that with `'try'` invalid authentication is accepted, and the user will still reach the route handler.
+
+When specifying one strategy, you may set the `strategy` property to a string with the name of the strategy. When specifying more than one strategy, the parameter name must be `strategies` and should be an array of strings each naming a strategy to try. The strategies will then be attempted in order until one succeeds, or they have all failed.
+
+Lastly, the `payload` parameter can be set to `false` denoting the payload is not to be authenticated, `'required'` or `true` meaning that it *must* be authenticated, or `'optional'` meaning that if the client includes payload authentication information, the authentication must be valid.
+
+The `payload` parameter is only possible to use with a strategy that supports the `payload` method in its scheme.
+
+## <a name="basic" /> hapi-auth-basic
+
+The first `scheme` we will look at is the [hapi-auth-basic](https://github.com/hapijs/hapi-auth-basic) plugin.  Just like the name says, the `hapi-auth-basic` plugin uses basic authentication to validate users.  Here is an example of setting up `hapi-auth-basic`:
```

Comment:
There seem to be more spaces than needed before *Just like the name says* and before *Here is an example*